tests/scalers/mssql/mssql_test.go

```diff
@@ -4,7 +4,9 @@
 package mssql_test
```

Comment:
I think that we should split this file into 2, the old one just testing base mssql and another one within the azure folder to cover the workload identity integration. This will improve the readability of the test and also will help to detect the root cause of the failures (we did the same with prometheus e2e for example)